lib/alerts/components/AlertEditor.js

```diff
@@ -221,10 +220,16 @@ export default class AlertEditor extends Component<Props> {
                             : 'text-danger'
                         }
                         style={{fontWeight: 400}}>
-                        {titleCharactersRemaining}
+                        {titleCharacterCount}
                       </span>
                       <h5 style={{margin: '0px'}}>
                         <small>
+                          {titleCharacterCount > ALERT_TITLE_CHAR_LIMIT
```

Comment:
Suggestion:
```js
{titleCharacterCount > ALERT_TITLE_CHAR_LIMIT && (<span>...)}
```

Reply:
I think the ternary is OK (we use both ternary statements and `&&` throughout our code), but the part after the `:` should be `null` not `''` (empty string literal).

Reply:
I don't really see a point in using a ternary when nothing is shown in the false condition. I recommend using @binh-dam-ibigroup's suggestion.

Reply:
Good points. I think moving with more modern approaches is better and is less code too. Have changed it to &&